### PR TITLE
Upgrading terraform version in all-in-one image

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Changed terraform version from `v0.12.29` to `v0.13.6`.
Added steps for installing plugins `terraform-provider-ibm` v1.21.2 and `terraform-provider-null` v2.1.2.
Plugin path is as per https://www.terraform.io/docs/cli/config/config-file.html#implied-local-mirror-directories

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/179#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: images/all-in-one/Dockerfile
Comment (mkumatag) [inline images/all-in-one/Dockerfile:138]: Let's use the above environment variables for the versions

Comment (mkumatag) [inline images/all-in-one/Dockerfile:138]: Let's use the above environment variables for the versions

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: images/all-in-one/Dockerfile
Comment (mkumatag) [inline images/all-in-one/Dockerfile:7]: not required, you can remove this line.

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Review by mkumatag — APPROVED** [review]
/lgtm

will have to refactor the whole docker file with better optimization later,

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/179#pullrequestreview-639548335" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/179#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
